percy/snapshot.py

```diff
@@ -81,3 +82,29 @@ def percy_snapshot(driver, name, **kwargs):
     except Exception as e:
         print(f'{LABEL} Could not take DOM snapshot "{name}"')
         print(f'{LABEL} {e}')
+
+# Take screenshot on driver
+def percy_screenshot(driver, name, **kwargs):
+    if not is_percy_enabled(): return
+    if not isinstance(driver, WebDriver): return
```

Comment:
until function is not called we don't have access of driver in init. either we need to create separate function
Yes it will return false for appium driver

Reply:
oh correct, I was thinking we have something in `__init__.py` and delegate the call here, 

* lets throw module not found error here which tells users to install `percy-appium-app` package if Appium driver is passed
* if Non Web driver is passed - unsupported driver error

Reply:
To check if Appium driver is passed we need to install appium driver in this sdk. should we install appium or only throw unsupportedwebdriver if non selenium web driver

Reply:
Cam we make a network call to remote selenium server to get information?